adafruit_bitmapsaver.py

```diff
@@ -32,19 +33,25 @@
 import board
 from displayio import Bitmap, Palette, Display
 
+try:
+    from typing import Tuple, Optional, Union
+    from io import BufferedWriter
```

Comment:
I think maybe instead of `BufferedWriter` we may want `WriteableBuffer` from circuitpython_typing but I'm not 100% certain. @tekktrik do you think `WriteableBuffer` makes sense as used in this library?